aten/src/ATen/autocast_mode.cpp

```diff
@@ -400,6 +487,9 @@ TORCH_LIBRARY_IMPL(aten, Autocast, m) {
   KERNEL(ADD_NS(cudnn_convolution_transpose), "cudnn_convolution_transpose.deprecated", Tensor (const Tensor &, const Tensor &, const c10::optional<Tensor>&, IntArrayRef, IntArrayRef, IntArrayRef, IntArrayRef, int64_t, bool, bool), fp16)
   KERNEL(ADD_NS(cudnn_convolution), "cudnn_convolution", Tensor (const Tensor &, const Tensor &, IntArrayRef, IntArrayRef, IntArrayRef, int64_t, bool, bool), fp16)
   KERNEL(ADD_NS(cudnn_convolution_transpose), "cudnn_convolution_transpose", Tensor (const Tensor &, const Tensor &, IntArrayRef, IntArrayRef, IntArrayRef, IntArrayRef, int64_t, bool, bool), fp16)
+  m.impl("_cudnn_rnn",
+         c10::impl::hacky_wrapper_for_legacy_signatures<std::tuple<Tensor,Tensor,Tensor,Tensor,Tensor> (const Tensor &, TensorList, int64_t, const c10::optional<Tensor>&, const Tensor &, const c10::optional<Tensor>&, int64_t, int64_t, int64_t, bool, double, bool, bool, IntArrayRef, const c10::optional<Tensor>&)>(TORCH_FN(at::autocast::_cudnn_rnn_cast_reflatten))
```

Comment:
No, it shouldn't be necessary to use `hacky_wrapper_for_legacy_signatures`. cc @smessmer

Reply:
Please don't use the hacky_wrapper. Maybe I have to make the name even more scary to prevent people from using it :P

Are you rebased on current master? You shouldn't need the hacky wrapper here if your PR base is recent enough.

Reply:
recent-ish master. autogenned code in VariableTypeWhatever.cpp used `hacky_wrapper_for_legacy_signatures` for _cudnn_rnn.  will rebase and try without it.

Reply:
@smessmer see, this is why you should fix earlier rather than later. People still looking at codegen to figure out what to do ;)

Reply:
hopefully resolved